### PR TITLE
Batch LR buys

### PR DESCRIPTION
**Pull request by hensha256**
Also make Merkle fields optional given API returns null if they dont exist

### CODE REVIEW

File: src/entities/protocols/looksRareV2.ts [hunk 1]
```diff
@@ -101,29 +112,37 @@ export class LooksRareV2Trade extends NFTTrade<LooksRareV2Data> {
...
-    const takerBid: TakerOrder = {
-      recipient: data.taker,
-      additionalParameters: '0x',
-    }
+      makerSignatures.push(order.apiOrder.signature)
 
-    const value: BigNumber = BigNumber.from(data.apiOrder.price)
+      takerBids.push({
+        recipient: order.taker,
+        additionalParameters: '0x',
```
Comment (zhongeric) [inline src/entities/protocols/looksRareV2.ts:135]: Is it possible that there are additionalParameters specified in the `order` object returned from the API?

File: test/nftTrades.test.ts [hunk 2]
```diff
@@ -108,6 +107,26 @@ describe('SwapRouter', () => {
...
+
+    it('encodes batch buying 2 ERC721s from LooksRare', async () => {
+      // buy items from block 17037140
+      const looksRareV2Data1: LooksRareV2Data = {
+        apiOrder: looksRareV2Orders[1],
+        taker: recipient,
+      }
+      const looksRareV2Data2: LooksRareV2Data = {
+        apiOrder: looksRareV2Orders[2],
+        taker: recipient,
```
Comment (zhongeric) [inline test/nftTrades.test.ts:119]: Can we get a test that has the zero address as the taker and ensure that the router sweeps the NFT back to the recipient after the trade?

**Review by zhongeric — APPROVED** [review]
Looks good to me, one request for additional test coverage if you agree!

File: src/entities/protocols/looksRareV2.ts (hunk 1 above)
Comment (hensha256) [reply src/entities/protocols/looksRareV2.ts:135]: the additional parameters are never actually used for anything onchain... have messaged LooksRare team to ask what theyre for

**Pull request merged**